app/forms/support_interface/conditions_form.rb

```diff
@@ -247,5 +252,41 @@ def structured_conditions_to_save
         )
       end
     end
+
+    def ske_conditions_are_valid
+      validate_ske_conditions
+      validate_language_count if language_course?
+      validate_combined_ske_length if multiple_ske_conditions?
+    end
+
+    def validate_combined_ske_length
+      ske_conditions_to_validate = structured_conditions_to_save
+      if ske_conditions_to_validate.many? && ske_conditions_to_validate.none? { |sc| sc.length == '8' }
+        errors.add(:base, :must_have_at_least_one_8_week_ske_course)
+      end
+    end
+
+    def validate_language_count
+      if ske_conditions.length > SkeCondition::MAX_SKE_LANGUAGES
+        errors.add(:base, :too_many, count: SkeCondition::MAX_SKE_LANGUAGES)
+      end
+    end
+
+    def multiple_ske_conditions?
+      ske_conditions.length > 1
+    end
+
+    def validate_ske_conditions
+      structured_conditions_to_save.each_with_index do |ske_condition, index|
+        ske_condition.validate
+
+        ske_condition.errors.each do |error|
+          next if error.attribute == :offer
```

Comment:
what does `error.attribute == :offer` indicate?

Reply:
This is originally from `OfferWizard`. I guess its there because we want to validate a detached `SkeCondition` so we want to ignore the presence validation on the `offer` association.